Website/src/items/powershell-scripts/EdgeWebsite/DAM Image Inventory.yml

```diff
@@ -0,0 +1,110 @@
+﻿---
+ID: "b5eb5a74-da3a-4b26-8ee9-5b2c8db65d34"
+Parent: "6053403f-2ebd-4546-bc1c-869bbb66c320"
+Template: "dd22f1b3-bd87-4db2-9e7d-f7a496888d43"
+Path: /sitecore/system/Modules/PowerShell/Script Library/SPE/Reporting/Content Reports/Reports/EdgeWebsite/DAM Image Inventory
+SharedFields:
+- ID: "06d5295c-ed2f-4a54-9bf2-26228d113318"
+  Hint: __Icon
+  Value: Office/32x32/revert_database.png
+- ID: "b1a94ff0-6897-47c0-9c51-aa6acb80b1f0"
+  Hint: Script
+  Value: |
+    $ContentHubHostname = 'mint.stylelabs.io'
```

Comment:
Having "mint" hardcoded in this variable will make detection to fail once we start using other ContentHub sandboxes like "demoedge". Maybe just keeping ".stylelabs.io" would be enough?